### PR TITLE
Promote ShareProcessNamespace to beta in docs

### PR DESCRIPTION
**Pull request by verb**
PR requested here https://github.com/kubernetes/features/issues/495#issuecomment-414452158

Nothing has changed for this feature, only a promotion from alpha to beta.

### CODE REVIEW

File: content/en/docs/tasks/configure-pod-container/share-process-namespace.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ include debugging utilities like a shell.
 
 {{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
 
-A special **alpha** feature gate `PodShareProcessNamespace` must be set to true
+A special **beta** feature gate `PodShareProcessNamespace` must be set to true
```
[the remark this answers is not in the document]
Comment (tengqm) [reply content/en/docs/tasks/configure-pod-container/share-process-namespace.md]: right. beta feature gates are supposed be enabled by default. Please clarify.

**Comment by tengqm** [comment]
pls also fix the feature-gates.md file.

File: content/en/docs/tasks/configure-pod-container/share-process-namespace.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (verb) [reply content/en/docs/tasks/configure-pod-container/share-process-namespace.md]: Yes, it's true by default. I looked around for standard language here and didn't find anyway, so I just explained explicitly. PTAL.

**Comment by verb** [comment]
@zparnold you mean beside the doc in `docs/tasks/configure-pod-container/share-process-namespace.md` included in this PR?

I'm happy to write more docs for this if you point me in a good direction. When I asked what docs I should write for Alpha someone told me to write a task doc. It didn't occur to me to ask if there were additional doc requirements for Beta.

**Comment by zparnold** [comment]
@verb I don't think there is an additional requirement, I just know that alpha features have the ability to mutate pretty significantly without warning. If you don't believe there's anything else to do here I'm ready to merge.

**Comment by verb** [comment]
@zparnold Ahh, apologies, I misread your initial comment. No, this only needs an update to feature status. The feature didn't change at all from alpha to beta. Thanks!

**Comment by zparnold** [comment]
Sounds good!
/approve

**Pull request merged**